.github/workflows/schemaspy.yml

```diff
@@ -25,17 +25,23 @@ jobs:
       - uses: actions/setup-node@master
         with:
           node-version: 16
+      - run: |
+          sudo apt-get install --yes --no-install-recommends postgresql-client
+      - run: |
+          export PGPASSWORD='postgres';
+          psql -h 127.0.0.1 -U postgres -c 'CREATE SCHEMA IF NOT EXISTS portal;'
```

Comment:
Why do we need to create portal schema here?

Reply:
So that it exists. The default postgres image has public created, and currently our migration scripts creates ALCS, but once we flatten the migration scripts to use only ALCS this will need to be updated to also create ALCS. Without this line the portal migrations will fail with something like, schema portal does not exist.

Reply:
I thought we already doing that in scope of postgres-schema.sql in docker folder